content/en/sidecar-container.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Sidecar Container
+status: Feedback Appreciated
+category: concept
+---
+
+The Sidecar Container is an implementation of the sidecar pattern. An application deployed on a second container will run side by side and share its lifecycle with a primary application running on a main container.
+
+## Problem it addresses
+
+There are scenarios where it is convenient to group [containers](/container/) to deal with problems related to the need to support cross-platform requirements or shared functionality such as logging, monitoring, tracing or [observability](/observability/) in general, and security, traffic management, networking and transversal application configuration. These common functionalities can be implemented independently as services and run on separate containers.
```

Comment:
IMHO: plz clarify a bit more about what "transversal application configuration" refers to.

Reply:
Clarified. Thanks @Okabe-Junya